libs/dh/shared/domain/src/lib/permission.ts

```diff
@@ -14,9 +14,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-export type Permission =
-  | 'organization:manage'
-  | 'gridareas:manage'
-  | 'actor:manage'
-  | 'users:manage';
+export const permissions = [
+  'organization:manage',
+  'gridareas:manage',
+  'actor:manage',
+  'users:manage',
+] as const;
+export type Permission = typeof permissions[number];
```

Comment:
praise: nice change.

question: not that it should be done, but could this also be in a `domain` library under `dh/auth`?

Reply:
I don’t see anything should be wrong doing that at a later point :)